server/src/main/java/org/opensearch/gateway/remote/ClusterMetadataManifest.java

```diff
@@ -411,24 +542,52 @@ public ClusterMetadataManifest(StreamInput in) throws IOException {
             this.globalMetadataFileName = null;
             this.routingTableVersion = in.readLong();
             this.indicesRouting = Collections.unmodifiableList(in.readList(UploadedIndexMetadata::new));
-        } else if (in.getVersion().onOrAfter(Version.V_2_12_0)) {
```

Comment:
After the backport of split PR in 2.x, we didnt update the version check at line no. 534 to 2.15.0. Can you please ensure version is updated in main as well after backport is merged in 2.x

Reply:
I'll raise a separate PR for updating the check.